geps/gep-713.md

```diff
@@ -38,24 +58,210 @@ portable.
 * Define all potential policies that may be attached to resources
 * Design the full structure and configuration of policies
 
+## Background and concepts
+
+When designing Gateway API, one of the things we’ve found is that we often need to be
+able change the behavior of objects without being able to make changes to the spec
+of those objects. Sometimes, this is because we can’t change the spec of the object
+to hold the information we need ( ReferenceGrant, from GEP-709, affecting Secrets
+and Services is an example, as is Direct Policy Attachment), and sometimes it’s
+because we want the behavior change to flow across multiple objects
+(this is what Hierarchical Policy Attachment is for).
+
+To put this another way, sometimes we need ways to be able to affect how an object
+is interpreted in the API, without representing the description of those effects
+inside the spec of the object.
+
+This document describes the ways we design objects to meet these two use cases,
+and why you might choose one or the other.
+
+We use the term “metaresource” to describe the class of objects that _only_ augment
+the behavior of another Kubernetes object, regardless of what they are targeting.
```

Comment:
I'd argue that you're still augmenting the _behavior_, because the underlying object is _part of_ the behavior.

Also, the word `augment` took a lot of arguing to get to, so I'm reluctant to mess with this too much. 😄